conf/app.routes

```diff
@@ -40,3 +40,7 @@ GET         /update-remove-website-address     controllers.website.ConfirmRemove
 # capture telephone numbers
 GET         /new-telephone-numbers             controllers.contactNumbers.CaptureContactNumbersController.show
 POST        /new-telephone-numbers             controllers.contactNumbers.CaptureContactNumbersController.submit
+
+# confirm phone numbers
+GET         /confirm-new-telephone-numbers     controllers.contactNumbers.ConfirmPhoneNumbersController.show
+POST        /confirm-new-telephone-numbers     controllers.contactNumbers.ConfirmPhoneNumbersController.updatePhoneNumbers
```

Comment:
Our other journeys use `GET` routes for the update functions. I think that's because in the past we had difficulty with the performance tests hitting the route as a POST. Are there any benefits to having it as a POST route over a GET?

Reply:
Hmm - I guess it's just the way i've always done it - GET to show the page and the button (Although right now not a button but rather an anchor tag) POST on the same URL. I'll swap to GET for consistency's sake.